doc/cephadm/services/osd.rst

```diff
@@ -1,7 +1,6 @@
 ***********
 OSD Service
 ***********
-.. _device management: ../rados/operations/devices
```

Comment:
I'm pretty sure the fix would be to write

`../../rados/operations/devices`

as opposed to

`../rados/operations/devices`

Reply:
Ah, I see @zdover23 advised to add an internal link as described in https://docs.ceph.com/en/reef/start/documenting-ceph/#internal-links, which is what the current fix is attempting. Perhaps you have further advice on this method @zdover23 ?